Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ExchangeManagerBean.java

```diff
@@ -324,11 +316,8 @@ public boolean getDisableButtonsEndpoint() {
     }
 
     public boolean isRefreshLocked() {
-        if (recordRefreshLocked != exchangeService.isRefreshLocked()) {
-            recordRefreshLocked = exchangeService.isRefreshLocked();
-            RequestContext.getCurrentInstance().update(Arrays.asList(BUTTONS_REFRESH_LOCKED));
-        }
-        return exchangeService.isRefreshLocked();
+        // web-service no-longer return refreshLocked; action will fail if the exchangeManager is refreshLocked
```

Comment:
if this is the case, why cant we strip out this function and the calls to this method entirely?

Reply:
GUI interface still used the method so false required to return